doc/breakages.md

```diff
@@ -1,12 +1,22 @@
 # Minetest Major Breakages List
 
 This document contains a list of breaking changes to be made in the next major version.
+This list is largely advisory and items may be reevaluated once the time comes.
 
-* Remove attachment space multiplier (*10)
-* Remove player gravity multiplier (*2)
+* remove attachment space multiplier (*10)
+* remove space multiplier for models (*10)
+* remove player gravity multiplier (*2)
 * `get_sky()` returns a table (without arg)
 * `game.conf` name/id mess
 * remove `depends.txt` / `description.txt` (would simplify ContentDB and Minetest code a little)
-* rotate moon texture by 180°, making it coherent with the sun (see https://github.com/minetest/minetest/pull/11902)
+* rotate moon texture by 180°, making it coherent with the sun
```

Comment:
Rotating the moon was already rejected once as being unnecessary, just breaking things.

The only thing that it does is break stuff, which can be unbrokes by rotating the texture again.

IIRC @Zughy was super-in-favor of it for some reason, no one else had actually thought it even a bug.